modules/usbip.nix

```diff
@@ -21,7 +21,7 @@ in
 
     snippetIpAddress = lib.mkOption {
       type = lib.types.str;
-      default = "$(ip route list | sed -nE 's/(default)? via (.*) dev eth0 proto kernel/\2/p')";
+      default = "$(ip route list | sed -nE 's/(default)? via (.*) dev .*/\\2/p' | head -n1)";
```

Comment:
I think we want to keep the eth0 and so on and just add the wildcard after it

Reply:
What about the `head -n1`, otherwise you risk getting multiple matches, which won't work

Reply:
Any reason to keep it at eth0 by the way? If not wsl-vpnkit would work out of the box.

Reply:
@SuperSandro2000 Pinging in case you didn't see my follow-up questions.

Reply:
> What about the `head -n1`, otherwise you risk getting multiple matches, which won't work

yeah, lets add that.

> Any reason to keep it at eth0 by the way? If not wsl-vpnkit would work out of the box.

Thats the default interface WSL uses.

Reply:
@SuperSandro2000 I adjusted it per your request. I am still not sure what the drawback is to not specify eth0 as it will work in all the cases. Instead of when you do specify it, those users won't have it working out of the box, but will have to adjust this configuration (which is not a big deal, but still).